test_runner/src/main/kotlin/ftl/presentation/cli/firebase/test/networkprofiles/NetworkProfilesDescribeCommand.kt

```diff
@@ -32,4 +36,12 @@ class NetworkProfilesDescribeCommand :
     override var profileId: String = ""
 
     override fun run() = invoke()
+
+    override val out = outputLogger {
+        when (this) {
+            is NetworkProfile -> prepareDescription()
+            is String -> this
```

Comment:
```suggestion
```
1. We shouldn't handle types that we do not expect.
2. IMO `String` is too general to be used as output type, We want more specialized types here.